deployment/hasura/metadata/databases/AerieMerlin/tables/metadata/plan_snapshot_tags.yaml

```diff
@@ -0,0 +1,48 @@
+table:
+  name: plan_snapshot_tags
+  schema: metadata
+configuration:
+  custom_name: "plan_snapshot_tags"
+object_relationships:
+  - name: plan_snapshot
+    using:
+      foreign_key_constraint_on: snapshot_id
+  - name: tag
+    using:
+      foreign_key_constraint_on: tag_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [snapshot_id, tag_id]
+      check: {}
+  - role: user
+    permission:
+      columns: [snapshot_id, tag_id]
```

Comment:
Are we explicitly listing all columns instead of using `"*"` here in anticipation of other columns being added?

Reply:
Technically, however on this table specifically it was mainly done in order to maintain a consistent style across all of the insert/update permissions in our metadata (that style being that insert/update permissions are **explicitly** granted rather than **implicitly** given via the `"*"`). This consistency is especially important as we expect users to actually view these files and use them as reference when making their own roles.